afctl/parser_helpers.py

```diff
@@ -95,9 +95,14 @@ def generate_config_file(files):
     @staticmethod
     def generate_project(args, files):
         try:
+            flag = False
             if args.name != '.':
-                os.mkdir(files['main_dir'])
-                ParserHelpers.init_project(files)
+                if not os.path.exists(files['main_dir']):
```

Comment:
if exist, does it copy the content or ignore?

Reply:
It will ignore everything. It will show error that project generation fail.